features/networking/sdn.feature

```diff
@@ -220,11 +218,11 @@ Feature: SDN related networking scenarios
     And evaluation of `pod(1).name` is stored in the :pod2_name clipboard
 
     # Kill ovs process
-    When I run command on the "<%= cb.node_name %>" node's sdn pod:
-      | bash | -c | pgrep ovs-vswitchd |
+    When I run command on the "<%= cb.node_name %>" node's ovs pod:
+      | pgrep ovs-vswitchd |
     Then the step should succeed
-    When I run command on the "<%= cb.node_name %>" node's sdn pod:
-      | bash | -c | pkill ovs-vswitchd |
+    When I run command on the "<%= cb.node_name %>" node's ovs pod:
+      | pkill ovs-vswitchd |
```

Comment:
Even with this fix OCP-25706  fails as expected due to https://bugzilla.redhat.com/show_bug.cgi?id=1854801